privacyidea/webui/login.py

```diff
@@ -134,6 +134,17 @@ def single_page_application():
     else:
         customization_baseline_file = "templates/baseline.html"
 
+    login_text = policy_object.get_action_values(
+        allow_white_space_in_action=True,
+        action=ACTION.LOGIN_TEXT,
+        scope=SCOPE.WEBUI,
+        client=client_ip, unique=True
+    )
+    if len(login_text) and list(login_text)[0] and sub_state not in [1, 2]:
+        login_text = list(login_text)[0]
```

Comment:
Could you add a unit test for this code path? Maybe just something simple, e.g. changing the login text to "Hello there" and checking if "Hello there" is part of the returned HTML.

Reply:
OK. Will do.

Reply:
done

Reply:
:+1: thanks!